client/blocks/nps-survey/index.jsx

```diff
@@ -155,13 +196,64 @@ class NpsSurvey extends Component {
 						className="nps-survey__not-answer-button"
 						onClick={ this.handleFeedbackFormClose }
 					>
-						{ translate( 'Close' ) }
+						{ this.shouldShowPromotion() ? translate( 'Skip' ) : translate( 'Close' ) }
 					</Button>
 				</div>
 			</Fragment>
 		);
 	}
 
+	renderPromotion() {
+		const { hasAvailableConciergeSession, translate } = this.props;
+
+		return (
+			<div className="nps-survey__promotion">
+				<p>{ translate( 'Thank you for your feedback. We’d like to help!' ) }</p>
+				{ hasAvailableConciergeSession && (
+					<Fragment>
+						<p>
+							{ translate(
+								'You have a free, 45-minute one-on-one call with a website expert as part of your WordPress.com Business plan benefits.'
+							) }
+						</p>
+						<p>
+							{ translate(
+								'{{booking}}Reserve a 1:1 Support Session{{/booking}} now or connect with the support team {{contact}}over live chat or email{{/contact}}.',
+								{
+									components: {
+										booking: <a href="/me/concierge" onClick={ this.handleLinkClick } />,
+										contact: <a href={ CALYPSO_CONTACT } onClick={ this.handleLinkClick } />,
+									},
+								}
+							) }
+						</p>
+					</Fragment>
+				) }
+				{ ! hasAvailableConciergeSession && (
+					<p>
+						{ translate(
+							'Connect with the WordPress.com support team {{contact}}over live chat or email{{/contact}} right now and we’d love to help you as best we can.',
```

Comment:
Let's actually remove that part of the copy entirely because it's redundant to what we say earlier. So that section should read: "Connect with our WordPress.com support team over live chat or email now."
Please read carefully when making the changes because I made a couple of minor tweaks to that sentence too.